### PR TITLE
Sdc/rm rebar2

### PR DESCRIPTION
**Pull request by spencerdcarlson**
I was iterating on the `rebar.config` and `rebar.config.script` and realized that the CI is always using rebar3, even with OTP 23.3.4.7 so I think we can clean it up a bit.  This MR will be a lot easier to review once https://github.com/kafka4beam/brod/pull/499 is merged.
Screen Shot 2022-05-06 at 3 56 13 PM

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -8,14 +8,54 @@
...
+        {copy, "{{lib_dirs}}/crc32cer/priv/crc32cer*.so", "bin"},
+        {copy, "{{lib_dirs}}/snappyer/priv/snappyer.so", "bin"}
+      ]}
+    ]}]},
+  {test, [
+    {deps, [ {docopt, {git, "https://github.com/zmstone/docopt-erl.git", {branch, "0.1.3"}}}
+           , {jsone, "1.7.0"}
+           , {meck, "0.9.2"}
+           , {proper, "1.4.0"}
+           , {snabbkaffe, "0.6.0"}
```
Comment (zmstone) [inline rebar.config:31]: Possible to use the latest version? 
```suggestion
           , {snabbkaffe, "1.0.1"}
```

Comment (spencerdcarlson) [reply rebar.config:31]: I tried to use the latest, but it caused some tests to fail. 

The commit before this used [{snabbkaffe, "1.0.1"}](https://github.com/kafka4beam/brod/pull/500/commits/1784009862d552aa7ac02292dcd44de01be3ac2e#diff-427d8ddce21ccc4fae24fb547c7a82c2a793ed57208d8343e519e0aec8119ff7R31) which caused the tests to fail. maybe I can look at trying to upgrade it in another MR.

Comment (zmstone) [reply rebar.config:31]: Alright, let’s handle this change separately

**Comment by zmstone** [comment]
Hi @spencerdcarlson 

Need a rebase

**Review by zmstone — APPROVED** [review]

**Comment by spencerdcarlson** [comment]
I opened https://github.com/klarna/snabbkaffe/issues/45 to address the snabbkaffe v1.0.1 issue. TLDR It looks like they need to publish the latest

**Comment by zmstone** [comment]
> I opened [klarna/snabbkaffe#45](https://github.com/klarna/snabbkaffe/issues/45) to address the snabbkaffe v1.0.1 issue. TLDR It looks like they need to publish the latest

snabbkaffe author is here in this org.
and it's forked in this org too, the real upstream is: https://github.com/kafka4beam/snabbkaffe

**Pull request merged**